packages/livekit-server-sdk/src/IngressClient.ts

```diff
@@ -22,11 +22,11 @@ export interface CreateIngressOptions {
    */
   name?: string;
   /**
-   * name of the room to send media to. optional
+   * name of the room to send media to.
    */
   roomName?: string;
   /**
-   * unique identity of the participant. optional
+   * unique identity of the participant.
    */
   participantIdentity?: string;
```

Comment:
since they are not optional, we shouldn't make them `?`. and should move these to the top?

Reply:
Would that potentially break any existing code using the SDK?

Reply:
if this is required, then existing code would either have to set it, or it wouldn't be successful either way, right?